website/components/LandingPage/index.tsx

```diff
@@ -241,10 +247,11 @@ export default function LandingPage(): ReactElement {
   // TODO: Should the load buttons be link elements or buttons?
   // Currently both the link and the button inside can be tab-selected.
   const renderDataset = (dataset: DatasetEntry, index: number): ReactElement => {
+    // TODO: Use links here instead of button onClicks.
     return (
       <DatasetCard key={index}>
-        <h4>{dataset.name}</h4>
-        <p>{dataset.description}</p>
+        <h3>{dataset.name}</h3>
+        {dataset.description && <p>{dataset.description}</p>}
         <Button type="primary" onClick={() => onClickLoad(dataset.loadParams)}>
           Load<VisuallyHidden> dataset {dataset.name}</VisuallyHidden>
         </Button>
```

Comment:
Visual comment: I think the current layout of datasets could do a bit more to visually group titles to load buttons, and to differentiate the row of datasets from the publication description row. Don't know how much design this has been through, but might be worth talking to UX about introducing something like a mini version of the card layout used by the CFE landing page? Regardless, this is a minor bit of polish and, if it even needs addressing at all, can wait until after an initial publish.

Reply:
That makes sense, thanks for the feedback. I'll pass it along to Lyndsay during our next UX OH.